sdk/terraform/Azure.ResourceManager.Terraform/tests/Tests/ExportTerraformTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Terraform.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Terraform.Tests.Tests
```

Comment:
could we remove a segment `Tests` here? We have a duplicate

Reply:
Fixed in 00549315b60